File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.java
class Solution {
    public String[] solution(String[] strArr) {
        String[] answer = strArr;
        for(int i = 0; i< answer.length; i++){
           if(i % 2 != 0){
             answer[i] = answer[i].toUpperCase();  
           }else{
             answer[i] = answer[i].toLowerCase();  
           }
        }
        return answer;
    }
}

File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.js
function solution(strArr) {
    var answer = [];
    strArr.map((string,index)=>{
        if(index % 2 === 0){
          answer.push(string.toLowerCase())   
        }else if(index % 2 === 1){
        answer.push(string.toUpperCase())
    }})
    return  answer 
}